lib/crawler.js

```diff
@@ -75,6 +75,7 @@ async function _retrievePackageVersion({
     version,
     outputPrefix = ''
 }) {
+    version = version || version.version;
```

Comment:
when does `version.version` exist?

Reply:
i probably had some funky package.json file at my work place when i tested, i've just tested with my home machine, they shouldn't occur. I've followed up with another commit and removed this line